src/main/java/com/google/sps/tool/WordSearch.java

```diff
@@ -0,0 +1,104 @@
+package com.google.sps.tool;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.Scanner;
+import java.util.Collections;
+import java.util.Random;
+
+public class WordSearch {
+  private static HashMap<Character, ArrayList<String>> firstLetterMap =
```

Comment:
I think a better name could be used for firstLetterMap. It's a bit genric. maybe something with alphabet?

Reply:
done.